### ISSUE TITLE
Problem with selecting the protocol when creating a user for visually impair people

### ISSUE THREAD

**Issue by MH140000** [Gozargah/Marzban#297]
**Describe the bug**
When creating a user, when a blind person uses a screen reader, he cannot select protocols or unselect it. Please solve this problem

**To Reproduce**
1. Go to 'marzban panel'
2. Click on 'creat user'
3. Move through the options with the cursor or tab keys
4. You will notice that you cannot move on different protocols with the keyboard

**Expected behavior**
It is expected that, like the account expiration date selection section, it is possible to select or unselect of protocols with the keyboard, and to label the protocols so that a blind person can manage them.

**Comment by erfjab** [comment]
Hello and good time. Do you still have this need?

**Comment by MH140000** [comment]
Hello. Yes. I stopped using Marzban because of this problem.

**Comment by erfjab** [comment]
> Hello. Yes. I stopped using Marzban because of this problem.

I will raise this issue in the team. I hope I can give you and other blind friends some good news. Accept our apologies for being too late to respond.

**Comment by MH140000** [comment]
Hello. Thank you. Please post here after solving the problem so I can check.
